understanding/low-or-no-background-audio.html

```diff
@@ -143,55 +134,48 @@ <h1><span class="standalone-resource__type-of-guidance">Understanding
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Prerecorded speech is not disrupted by background sound.</dd>
+                  <dt>目標</dt>
+                  <dd>収録済の発話が、背景音によって妨げられないようにする。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Avoid or lessen background sound, or let users turn it off.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>背景音を避けるもしくは小さくする、又は利用者が消せるようにする。</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>People who are hard of hearing may have difficulty distinguishing speech from music
-                     and other sounds.
+                  <dt>なぜそれが重要か</dt>
+                  <dd>音声の聞こえづらい利用者は、発話と音楽及びその他の音を、識別しづらい可能性がある。
```

Comment:
他では people を「人」としていたように思います。
「識別」というと、どうも大袈裟な感じがします。ここでは「区別」あるいは「聞き分ける」くらいで良いと思います。
may も「ことがある」くらいで良いように思いました。
「発話と音楽及びその他の音を」はどこで切れるのかわかりにくいので、「音楽及びその他の音を、発話と」のようにしてみると読みやすいかもしれません。
というところでこのようにしてはどうでしょうか?

```suggestion
                  <dd>音声の聞こえづらい人は、音楽及びその他の音を、発話と聞き分けづらいことがある。
```

Reply:
ありがとうございます。たしかに、people は「利用者」ではなく「人」ですね。distinguish の訳も、この文脈では「聞き分ける」が適切だと思います。